randovania/data/json_data/prime3.txt

```diff
@@ -3202,6 +3958,9 @@ Asset id: 6708692299270885612
   > Dock to Broken Lift
       Use Screw Attack (Space Jump)
 
+> Event - Gears Destroyed; Heals? False
+  * Event Gearworks Gears Destroyed
+
```

Comment:
Gearworks needs to have the doors connect to the pickup with sa and after gears destroyed.

The gears get destroyed after the fed landing site item I believe. So have the doors connect to the event with after fed landing site, and then the event connects to both doors as trivial.

Reply:
Connecting the Gearworks event as you described

If we set the gearworks to connect to both doors trivially, then wouldn't that mean logic would let you cross this room without SA if you've unlocked the fed landing site?

Reply:
Ah that's true. Having the event connect to only one door as Trivial would be better then, let's say connecting to only Broken Lift. That way SA is still required to cross the room.